src/Contexts/dataProvider.js

```diff
@@ -0,0 +1,91 @@
+import React, { createContext, useContext, useReducer, useState } from "react";
+import { checkIfProductExistsInList } from "../util";
+import { dataReducer } from "./dataReducer";
+
+export const DataContext = createContext();
+export function useData() {
+  return useContext(DataContext);
+}
```

Comment:
Isn't this a redundant export since we are already exporting useData?

Reply:
Yes it was unnecessary, I have removed it.